src/main/java/cartPage/CartPage.java

```diff
@@ -0,0 +1,24 @@
+package cartPage;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import pages.BasePage;
+
+public class CartPage {
+
```

Comment:
I think this page should have other classes from this package as parameters?

Reply:
Ok, done.